delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py

```diff
@@ -136,6 +141,15 @@ def _picking_data(self):
     def _get_carrier(self):
         return self.env.ref("delivery_carrier_label_ups.carrier_ups")
 
+    def test_picking_total_price(self):
+        """ Check total contracted price. Negotiated rates
+        have the priority over the published rates.
+         """
+        self._create_order_picking()
+        self.assertTrue(self.picking)
+        self.assertNotEqual(self.picking.carrier_price, 42.00)
+        self.assertEqual(self.picking.carrier_price, 39.89)
```

Comment:
I think this is sufficient, it's implied then that it's also not equal so some other number ;-)

Reply:
Yes, I just thought this way it would be more self-explanatory. But ok, I'll remove it.